OnContractChange/ipa.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Utility functions to read data from IPA Open Data (CSV formatted).
+ */
+import { pipeline } from "stream/promises";
+import { Transform } from "stream";
+import { parse } from "csv-parse";
+
+export type IpaCode = string;
+export type FiscalCode = string | undefined;
+export type IpaOpenData = ReadonlyMap<IpaCode, FiscalCode>;
+
+export const readIpaData = async (
+  stream: NodeJS.ReadableStream
+): Promise<IpaOpenData> => {
+  const ipaCode2FiscalCode = new Map<string, string | undefined>();
```

Comment:
@balanza I added `undefined` just because the type checker showed me an error 'cause I declared a non-optional field that used the result of `Map.get()` fn, which by definition can return undefined, but now I make that field optional, so if i remove it now there is no problem. So i will remove it, ok?

Reply:
resolved with a388e47409b9e1e1c759d17d8f9066b1a89309b9